client/state/application/actions.js

```diff
@@ -35,3 +47,36 @@ export function connectionRestored( noticeText ) {
 		dispatch( { type: CONNECTION_RESTORED } );
 	};
 }
+
+export function cssBuildFailed( noticeText ) {
+	return ( dispatch ) => {
+		dispatch( removeNotice( 'cssBuilding' ) );
+		dispatch( errorNotice(
+			noticeText
+			, {
```

Comment:
Sorry for the nitpick here, but we don't use comma-first style. Would you mind replacing with:

```js
noticeText, {
	…
} );

// or
dispatch( errorNotice(
	noticeText,
	{
		…
	}
)
```